seed/seed.ts

```diff
@@ -0,0 +1,49 @@
+import { dbConnection } from "../mongoConnection";
+import { addPlace, getAllPlaces } from "../data/places";
+
+async function main() {
```

Comment:
If this is for testing we should move the file to a tests folder

Reply:
This is for seeding, not testing. I didn't create any test cases yet because these db functions are temporary until we start writing the actual functions we'd need. Unless that would be the same thing? I never interpreted a seed to be a test, but I might just be looking at this from a different lens. lmk what you think!